### PR TITLE
Review of Developper install + Installing your Posgressql server and chill database

### PR DESCRIPTION
**Pull request by jphuart**
I've installed a developper version of Chill. I wanted also to have my own postgressql database to avoid to rely on a docker container that I don't manage well at the moment.

Here are the results of such exercice, and a try to complete the documentation.

### CODE REVIEW

File: source/installation/install_postgres_server.rst [hunk 1]
```diff
@@ -11,16 +11,64 @@
...
-.. todo::
+On a linux environment, installing Postgresql server is very easy.
+Here follows the instructions for a debian based distribution (as Ubuntu).
 
-   the section "Install PostresSql database" must be written. Help appreciated :-)
-   
+.. code-block:: bash
+	
+	sudo apt-get update
+	sudo apt-get install postgresql
```
Comment (julienfastre) [inline source/installation/install_postgres_server.rst]: It is recommanded to use the `apt` of postgresql. You will have always the last version. Doc here : https://wiki.postgresql.org/wiki/Apt

File: source/installation/install_postgres_server.rst [hunk 2]
```diff
@@ -11,16 +11,64 @@
...
+Here follows as an example the instructions that has been used on Ubuntu 14.04 distribution to install the Chill database.
+Feel free to customize it following your preferences, but do not forget to enable the `unaccent` extension on your database.
+
+.. code-block:: bash
 
-   Installing unaccent extension on ther server is possible with the package `postgresql-contrib-9.x` (replace 9.x with your server version). The extension may be enabled with running `CREATE EXTENSION unaccent;` in the database, with a superuser account.
+	sudo su
+	su postgres
+	psql
+		CREATE USER chill_user WITH PASSWORD 'my_terrible_secret' CREATEDB;
```
Comment (julienfastre) [inline source/installation/install_postgres_server.rst]: 'my terrible secret' : :smile:

Comment (julienfastre) [inline source/installation/install_postgres_server.rst]: It is easier to do this : 

```
sudo su
su postgres
createuser --pwprompt username
createdb -O username dbname
#then install unaccent
psql -d dbname -c "CREATE EXTENSION unaccent;"
```

But the instructions you wrote are ok.

If you think it is easier to do everything in SQL, you do not have to exit psql and connect again to switch to the newly created db. Simply use `\c chill_db` to connect to the chill_db database.

**Comment by julienfastre** [comment]
Thanks, @jphuart . For me it is ok, except that I think preferable to use the apt from postgresql. Could you do the change before I merge ?

**Comment by jphuart** [comment]
Ok I will do the changes you are asking for.

**Pull request merged**